packages/sprinkles/README.md

```diff
@@ -90,29 +90,25 @@ Create an `atoms.css.ts` file, then configure and export your `atoms` function.
 import { createAtomicStyles, createAtomsFn } from '@vanilla-extract/sprinkles';
 
 const space = {
-  none: 0,
-  small: 4,
-  medium: 8,
-  large: 16
-};
-
-const colors = {
-  blue50: '#eff6ff',
-  blue100: '#dbeafe',
-  blue200: '#bfdbfe',
+  0: '0',
+  4: '4px',
+  8: '8px',
+  12: '12px',
   // etc.
 };
 
-const layoutStyles = createAtomicStyles({
+const responsiveStyles = createAtomicStyles({
   conditions: {
     mobile: {},
     tablet: { '@media': 'screen and (min-width: 768px)' },
     desktop: { '@media': 'screen and (min-width: 1024px)' }
   },
   defaultCondition: 'mobile',
   properties: {
-    display: ['none', 'block', 'flex'],
+    display: ['none', 'flex', 'block', 'inline'],
     flexDirection: ['row', 'column'],
+    justifyContent: ['stretch', 'flex-start', 'center', 'flex-end'],
```

Comment:
Minor thing, I'd put these in as they are quite common and show the example of using extra values compared to alignItems
```suggestion
    justifyContent: ['stretch', 'flex-start', 'center', 'flex-end', 'space-around, 'space-between'],
```